doc/design/release.md

```diff
@@ -1,111 +1,45 @@
 # Steps to create a new release
 
-## Step 0: Review the Release Milestone
-
-If the release you plan to create corresponds with an existing [milestone](https://github.com/operator-framework/operator-lifecycle-manager/milestones/), make sure that all features have been committed. If a feature will not be added to the release be sure to remove it from the milestone.
-
-## Step 1: Installing Requirements
+The OLM project uses [GoReleaser](https://goreleaser.com/) to automatically produce multi-arch container images and release artifacts during the release process.
 
-Ensure you have `autoconf`, `automake`, and `libtool` installed.
-
-The following command can be used to install these packages on Fedora:
-
-```bash
-dnf install autoconf automake libtool
-```
+In order to create a new minor version release, simply create a new tag locally, push that tag up to the upstream repository remote, and let automation handle the rest.
 
-## Step 2: Verify Manifests
+The release automation will be responsible for producing manifest list container images, generating rendered Kubernetes manifests, and a draft release that will contain both of those attached as release artifacts.
 
-* Make sure you have a clean workspace. `git status` should show no change(s) or untracked file.
-* Make sure you pull the latest from `upstream`.
-* Checkout `master` branch.
-* Run `make release`
+## Step 0: Review the Release Milestone
 
-## Step 3: Bump the Version
+If the release you plan to create corresponds with an existing [milestone](https://github.com/operator-framework/operator-lifecycle-manager/milestones/), make sure that all features have been committed. If a feature will not be added to the release be sure to remove it from the milestone.
 
-* Bump the version in `OLM_VERSION` file. Make a new PR with this change only.
-* Wait until the PR has been merged.
+## Step 1: Setup the Release Tag
 
-## Step 4: Setup Tag
+In order to trigger the existing release automation, you need to first create a new tag locally, and push that tag up to the upstream repository remote.
 
-If git `push` is disabled on `upstream` repository in your fork, then clone this repository so that you can push to `master` directly.
+**Note**: The following steps assume that remote is named `origin`.
 
 * Pull the latest.
 * Make sure you are on `master` branch.
 * Make a new tag that matches the version.
 * Push tag directly to this repository.
 
 ```bash
-# 0.11.0 is the bumped version.
-git tag -a v0.18.0 -m "Version 0.18.0"
+# v0.20.0 is the bumped version.
+git tag -a v0.20.0 -m "Version 0.20.0"
```

Comment:
Not necessarily related to the scope of this PR, but... should we sign our tags? (with the `--sign` flag)

Reply:
I don't have any opposition towards doing that. Might be worth bubbling that up to the rest of the team to see if there are any strong objections.

Reply:
I'm for it.